src/i18n/de/vcard.json

```diff
@@ -93,7 +93,7 @@
     "message": "Alle akzeptieren"
   },
   "Search": {
```

Comment:
der Schlüssel 'Search' passt mit 'Kandidaten' nicht zusammen das führt später zu enormen Verwirrungen

Reply:
was ist mit der englischen Sprachdatei- Hier bleibt Search als Search?

Reply:
Ich fand Kandidaten besser als Suche, aber auch nicht 100% schön, hast du auch andere Vorschläge? Dann kann man alles anpassen.

Reply:
ich meinte eher das wenn der KEY "Search" heißt vermutet man dahinter den lokalisierten Text für Suche und nicht für Kandidaten - daher lieber einen neuen KEY anlegen statt einen bestehenden zu ändern. Kurzfristig macht das nichts aus, nur langfristig verwirrt man damit die Leute da der verwendete Key für Search plötzlich im deutschen Kandidaten ausspuckt statt Suche. Und bei solch einer Anpassung ist es dann auch wichtig in allen Sprachen entsprechend anzupassen.